src/ui/public/partials/saved_object_finder.html

```diff
@@ -1,27 +1,50 @@
-<form role="form">
-  <div class="form-group finder-form">
-    <div class="finder-form-options">
-      <a class="small" ng-click="finder.manageObjects(finder.properties.name)">manage {{finder.properties.nouns}}</a>
+<form role="form" class="form-inline">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="input-group form-group finder-form col-md-8">
+        <span class="input-group-addon">
+          <i class="fa fa-search"></i>
+        </span>
+        <input
+          input-focus
+          ng-model="filter"
+          ng-attr-placeholder="{{finder.properties.nouns}} Filter..."
+          ng-keydown="finder.filterKeyDown($event)"
+          class="form-control"
+          style="text-transform: capitalize"
+          name="filter"
+          type="text" />
+      </div>
+      <div id="finder-hit-count" class="col-md-2">
+        <span>{{finder.hitCount}} of {{finder.hitCount}}</span>
+      </div>
+      <div id="finder-manage-object" class="col-md-2">
+        <a class="small" ng-click="finder.manageObjects(finder.properties.name)">Manage {{finder.properties.nouns}}</a>
+      </div>
     </div>
-    <div class="clearfix visible-xs-block"></div>
-    <input
-      input-focus
-      ng-model="filter"
-      ng-attr-placeholder="{{finder.properties.noun}} Filter"
-      ng-keydown="finder.filterKeyDown($event)"
-      class="form-control"
-      name="filter"
-      type="text"
-      autocomplete="off" />
-
-    <span class="finder-hit-count"><strong>{{finder.hitCount}}</strong> {{finder.hitCountNoun()}}</span>
   </div>
 </form>
-<paginate list="finder.hits" per-page="{{perPage}}">
-  <ul
-    class="list-group list-group-menu"
+<paginate list="finder.hits" per-page="20">
+  <ul class="li-striped list-group list-group-menu"
     ng-class="{'select-mode': finder.selector.enabled}">
 
+    <li class="list-group-item list-group-menu-item">
+      <ul>
```

Comment:
This ul inside another ul doesn't feel right. It sort of feels like you're going for a table, but I don't think we need a table here either?